ecs/cloudformation.go

```diff
@@ -103,103 +73,31 @@ func (b *ecsAPIService) Convert(ctx context.Context, project *types.Project) ([]
 }
 
 // Convert a compose project into a CloudFormation template
-func (b *ecsAPIService) convert(project *types.Project) (*cloudformation.Template, map[string]string, error) { //nolint:gocyclo
-	var checker compatibility.Checker = &fargateCompatibilityChecker{
-		compatibility.AllowList{
-			Supported: compatibleComposeAttributes,
-		},
-	}
-	compatibility.Check(project, checker)
-	for _, err := range checker.Errors() {
-		if errdefs.IsIncompatibleError(err) {
-			logrus.Error(err.Error())
-		} else {
-			logrus.Warn(err.Error())
-		}
-	}
-	if !compatibility.IsCompatible(checker) {
-		return nil, nil, fmt.Errorf("compose file is incompatible with Amazon ECS")
-	}
-
+func (b *ecsAPIService) convert(project *types.Project) (*cloudformation.Template, error) { //nolint:gocyclo
```

Comment:
any chance to remove the //nolint now this has beed broken into smaller pieces ?